src/services/auth-service.ts

```diff
@@ -0,0 +1,48 @@
+import {CreateUserRequest, LoginRequest} from "../global";
+import {PrismaClient} from '@prisma/client'
+
+const prisma = new PrismaClient()
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const saltRounds = 10;
+
+export class AuthService {
+    registerUser = async (createUserRequest: CreateUserRequest) => {
```

Comment:
Because this is a service, `firstName` and `email` should be passed in as independent parameters, this couples the service too tightly to the request and means this method is not easily reusable.

If the intention that it wouldn't be reusable, making it as a service just adds more unnecessary abstraction